test/fuzz/tcti/tss2_tcti_fuzzing.h

```diff
@@ -0,0 +1,28 @@
+/* SPDX-License-Identifier: BSD-2 */
+/*
+ * Copyright (c) 2018, Intel Corporation
+ * All rights reserved.
+ */
+#ifndef TSS2_TCTI_FUZZING_H
+#define TSS2_TCTI_FUZZING_H
+
+#include "tss2_tcti.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+TSS2_RC tcti_platform_command(
```

Comment:
Is this function actually called an/or used anywhere ?
tcti_device also does not have a platform_command function. That one should be special for tcti_mssim only.